src/explore-education-statistics-common/src/modules/table-tool/components/SubjectForm.tsx

```diff
@@ -115,7 +113,13 @@ const SubjectForm = ({
           <Form {...form} id={formId} showSubmitError>
             <FormFieldRadioGroup<SubjectFormValues>
               name="subjectId"
-              legend={legend}
+              legend={[
+                legend,
+                <div key="legend-description" className="govuk-inset-text">
```

Comment:
Should use the `hint` prop. Having it like this isn't super accessible as it doesn't provide an association between it and the legend.

It's also weird to be using inset text here as well.